tutor/specs/e2e/assignment-review.e2e.ts

```diff
@@ -138,11 +138,11 @@ test.describe('Assignment Review', () => {
         const dueAtText = await page.$eval(
             '[data-test-id="due-date"]', node => (node as HTMLElement).innerText
         )
-        const newDueAt = DateTime.fromFormat(dueAtText + ' ' + timezone, reviewDateTimeFormat)
+        const newDueAt = DateTime.fromFormat(dueAtText, reviewDateTimeFormat)
```

Comment:
Is the timezone in dueAtText now? I had to add it manually before because it was elsewhere in the page.

Reply:
Yeah I replaced moment with Luxon in the Details render, so it displays a usable timezone (e.g. EST -> America/New_York): 
 ![](https://user-images.githubusercontent.com/34174/133332125-4414f628-ad77-4951-83b8-85f16c721a5a.png)